src/statistics/data.jl

```diff
@@ -66,3 +72,32 @@ function median_heuristic(d)
 
   min(m, l)
 end
+
+function mode_heuristic(d)
+
```

Comment:
```suggestion
```

Reply:
Also, it would be nice to add a comment with the link to the method that was implemented. It is a very nice resource for future readers.